Core/src/main/java/com/plotsquared/core/util/TabCompletions.java

```diff
@@ -78,29 +78,33 @@ private TabCompletions() {
      * cache, so it will complete will all names known to PlotSquared
      *
      * @param input    Command input
+     * @param issuer   The player who issued the tab completion
```

Comment:
Instead of breaking existing methods, deprecating and scheduling them for removal while pointing to the newer ones is preferred.

Reply:
Old methods are present again and deprecated